wgsl/index.bs

```diff
@@ -11443,16 +11457,24 @@ Note: The result is not mathematically meaningful when `abs(e)` &ge; 1.
   <tr algorithm="atan2">
     <td style="width:10%">Overload
     <td class="nowrap"><xmp highlight=rust>
-@const fn atan2(e1: T,
-                e2: T) -> T
+@const fn atan2(y: T,
+                x: T) -> T
 </xmp>
   <tr>
     <td style="width:10%">Parameterization
     <td>[ALLFLOATINGDECL]
   <tr>
     <td>Description
-    <td>Returns the prinicipal value, in radians, of the inverse tangent (tan<sup>-1</sup>) of `e1` &divide; `e2`.<br>
-    That is, approximates `x` with &pi;/2 &le; `x` &le; &pi;/2, such that `tan`(`x`) = `e1` &divide; `e2`.
+    <td>Returns an angle, in radians, in the interval [-&pi;, &pi;] whose tangent is `y`&divide;`x`.
```

Comment:
Suggest this be a link to angles:
```suggestion
    <td>Returns an [=angle=], in radians, in the interval [-&pi;, &pi;] whose tangent is `y`&divide;`x`.
```

If so, there are a few other instances worth updating.

Reply:
followup commit adds them to all instances.